api/envoy/config/core/v3/address.proto

```diff
@@ -102,6 +102,10 @@ message SocketAddress {
   // network_namespaces``). If this field is set, Envoy will create the socket in the specified
   // network namespace.
   //
+  // When configured for a listener, the network namespace filepath is also populated into the filter
+  // state with key ``envoy.network.network_namespace`` when a connection is accepted. This provides
+  // read-only access to the network namespace for filters, access logs, and other components.
+  //
```

Comment:
Sorry one comment..

I think it's more appropriate to have this comment on the socket address field inside the Listener proto. The actual `SocketAddress` proto here is used by other data planes (gRPC comes to mind) and this behavior is specific to Envoy.

Reply:
Done, Thanks!